addons/binding/org.openhab.binding.lgwebos/ESH-INF/binding/binding.xml

```diff
@@ -5,5 +5,14 @@
 	<name>LG webOS Binding</name>
 	<description>Binding to connect LG's WebOS based smart TVs based on Connect SDK</description>
 	<author>Sebastian Prehn</author>
+	
+	<config-description>
+        <parameter name="localIP" type="text">
+            <label>Local IP</label>
+            <description>Local IP of server network interface to bind to. This is optional. If not set, the binding will use openHAB's primary IP address, which may be configured under network settings.
+            </description>
```

Comment:
@htreu I would expect that we should also use the context network-address here?

Reply:
Yes, thats correct. It gives UIs a hint on how to render this field and optionally give input guidance.

Reply:
Added context network-address. But I don't see any change in Paper UI. I suspect that may be ok.

Reply:
Yes, Paper UI does not support special `network-address` parameter rendering right now. Thanks for adding it here, nonetheless.